pom.xml

```diff
@@ -318,19 +360,19 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>3.2.2</version>
+        <version>3.3.0</version>
         <configuration>
           <archive>
             <manifestEntries>
-              <Add-Opens>java.base/java.lang.reflect java.base/java.text java.base/java.util java.desktop/java.awt.font</Add-Opens>
+              <Add-Opens>java.base/java.util</Add-Opens>
```

Comment:
These additional `Add-Opens` directives were needed to support running on Java 11/17 on the old version of XStream we were previously depending on. Now that we depend on the latest Jenkins core weekly and therefore the latest version of XStream, only `java.base/java.util` is still needed. I tried removing that one, but I got XStream errors when testing in `jenkinsci/bom`, so keeping it.